<?xml version="1.0"?>
<launch>
	<!-- start control node -->
	<node pkg="controller" type="depth_controller.py" name="depth_controller" output="screen"/>
	<node pkg="controller" type="velocity_controller.py" name="velocity_controller" output="screen"/>
	<node pkg="controller" type="heading_controller.py" name="heading_controller" output="screen"/>
	<node pkg="controller" type="gamepad.py" name="gamepad" output="screen"/>
	<node pkg="interface" type="gui.py" name="gui" output="screen"/>
	<node pkg="controller" type="commander.py" name="commander" output="screen"/>
</launch>

